docs/Using-Fleet/REST-API.md

```diff
@@ -5119,38 +5119,36 @@ Returns the query specified by ID.
     "name": "centos_hosts",
     "description": "",
     "query": "select 1 from os_version where platform = \"centos\";",
+    "team_id": null,
+    "interval": 3600,
+    "platform": "",
+    "min_osquery_version": "",
+    "automations_enabled": true,
+    "logging": "snapshot",
     "saved": true,
     "observer_can_run": true,
     "author_id": 1,
     "author_name": "John",
     "author_email": "john@example.com",
-    "packs": [
-      {
-        "created_at": "2021-01-19T17:08:31Z",
-        "updated_at": "2021-01-19T17:08:31Z",
-        "id": 14,
-        "name": "test_pack",
-        "description": "",
-        "platform": "",
-        "disabled": false
-      }
-    ]
+    "packs": []
   }
 }
 ```
 
 ### List queries
 
-Returns a list of all queries in the Fleet instance.
+Returns a list of global queries or team queries.
 
 `GET /api/v1/fleet/queries`
 
 #### Parameters
 
-| Name            | Type   | In    | Description                                                                                                                   |
-| --------------- | ------ | ----- | ----------------------------------------------------------------------------------------------------------------------------- |
-| order_key       | string | query | What to order results by. Can be any column in the queries table.                                                             |
-| order_direction | string | query | **Requires `order_key`**. The direction of the order given the order key. Options include `asc` and `desc`. Default is `asc`. |
+| Name            | Type    | In    | Description                                                                                                                   |
+| --------------- | ------- | ----- | ----------------------------------------------------------------------------------------------------------------------------- |
+| order_key       | string  | query | What to order results by. Can be any column in the queries table.                                                             |
```

Comment:
by `query` we mean in the url right?

Reply:
Yes. I believe this is the standard way that it is referred to throughout the API docs.